service/src/main/java/org/cbioportal/genome_nexus/service/cached/CachedVariantIdAnnotationFetcher.java

```diff
@@ -0,0 +1,78 @@
+package org.cbioportal.genome_nexus.service.cached;
+
+import com.mongodb.DBObject;
+import org.cbioportal.genome_nexus.model.VariantAnnotation;
+import org.cbioportal.genome_nexus.persistence.VariantAnnotationRepository;
+import org.cbioportal.genome_nexus.persistence.internal.VariantAnnotationRepositoryImpl;
+import org.cbioportal.genome_nexus.service.exception.ResourceMappingException;
+import org.cbioportal.genome_nexus.service.transformer.ExternalResourceTransformer;
+import org.cbioportal.genome_nexus.service.remote.VEPDataFetcher;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.*;
+
+@Component
+public class CachedVariantIdAnnotationFetcher extends BaseCachedExternalResourceFetcher<VariantAnnotation, VariantAnnotationRepository>
+{
+    @Autowired
+    public CachedVariantIdAnnotationFetcher(ExternalResourceTransformer<VariantAnnotation> transformer,
+                                            VariantAnnotationRepository repository,
+                                            VEPDataFetcher fetcher,
+                                            @Value("${vep.max_page_size:200}") Integer maxPageSize)
+    {
+        super(VariantAnnotationRepositoryImpl.COLLECTION,
```

Comment:
this should prolly use a diff collection, since the response is slightly different

Reply:
hmm actually it seems to be the same response. I just had another look, so we might as well use the same collection